projects/kriya-dex-earn/index.js

```diff
@@ -1,35 +1,44 @@
 const { fetchURL } = require("../helper/utils");
 const sui = require("../helper/chain/sui");
 
-const clmmVaultUrl =
-  "https://vjkibmct3h.execute-api.ap-southeast-1.amazonaws.com/release/vaults";
-
-const llVaultUrl =
-  "https://c19eainr59.execute-api.ap-southeast-1.amazonaws.com/release/defilama/vaults";
-
-async function clmmVaultTVL(api) {
-  const vaults = (await fetchURL(clmmVaultUrl))?.data;
-  for (const vault of vaults) {
-    const tokenX = Number(vault?.coinA);
-    const tokenY = Number(vault?.coinB);
-
-    api.add(vault?.pool?.tokenXType, tokenX);
-    api.add(vault?.pool?.tokenYType, tokenY);
+const vaultsUrl =
+  "https://api.kriya.finance/defillama/vaults";
+
+const kdxStakingUrl =
+  "https://api.kriya.finance/kdx/metrics";
+
+const KDX_TYPE = "0x3b68324b392cee9cd28eba82df39860b6b220dc89bdd9b21f675d23d6b7416f1::kdx::KDX";
+const KDX_DECIMALS = 6;
+
+async function vaultTVL(api) {
+  try {
+    const vaults = (await fetchURL(vaultsUrl))?.data?.data;
+    for (const vault of vaults) {
+      if (vault?.vaultType === "KriyaClmm" || vault?.vaultType === "CetusClmm") {
```

Comment:
unsure how I missed this before, we should excluding KriyaClmm from tvl as we are already counting the dex part as tvl